lib/ts/recipe/session/sessionAuth.tsx

```diff
@@ -184,9 +179,7 @@ const SessionAuth: React.FC<PropsWithChildren<SessionAuthProps>> = ({ children,
         return null;
     }
 
-    return (
-        <SessionContext.Provider value={{ ...context, isDefault: false } as any}>{children}</SessionContext.Provider>
-    );
+    return <SessionContext.Provider value={context}>{children}</SessionContext.Provider>;
```

Comment:
why are we not giving isDefault as false anymore?

Reply:
We are checking for strict equality anyway and this way we don't need the `as any` to make the typing work.